models/intel/index.md

```diff
@@ -110,6 +110,9 @@ detect areas with complex shape (for example, free space on the road).
 | [road-segmentation-adas-0001](./road-segmentation-adas-0001/description/road-segmentation-adas-0001.md)                                        | 4.770                | 0.184      |
 | [semantic-segmentation-adas-0001](./semantic-segmentation-adas-0001/description/semantic-segmentation-adas-0001.md)                            | 58.572               | 6.686      |
 | [unet-camvid-onnx-0001](./unet-camvid-onnx-0001/description/unet-camvid-onnx-0001.md)                                                          | 260.1                | 31.03      |
+| [icnet-camvid-ava-tf-0001](./icnet-camvid-ava-tf-0001/description/icnet-camvid-ava-tf-0001.md)                                                   | 151.82                | 25.45       |
```

Comment:
Replace `ava-tf` with `tf-ava`

Reply:
fixed

Reply:
@eizamaliev why you request this change? Actual model name in internal infra ava-tf.